### ISSUE TITLE
Install problem with node 12

### ISSUE THREAD

**Issue by oliverjessner** [dachev/node-cld#54]
I am not C++ guy but i gues it's a node-gyp problem?

```
make: *** [Release/obj.target/cld/src/cld.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:200:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
```

**Comment by dachev** [comment]
@oliverjessner I just released 2.5.0. This should be fixed now.

**Comment by oliverjessner** [comment]
Thank you very much 💪
